src/components/ProductDetail.jsx

```diff
@@ -1,31 +1,50 @@
 import React, { useState, useEffect } from 'react';
-import { useParams, useHistory } from 'react-router-dom';
-import useFetch from '../hooks/useFetch';
+import { useParams, useNavigate } from 'react-router-dom';
+import { useFetch } from '../hooks/useFetch';
 import LoadingIndicator from './LoadingIndicator';
+import '../index.css';
 
 const ProductDetail = () => {
   const { id } = useParams();
-  const history = useHistory();
-  const { isLoading, data: product, error } = useFetch(`/api/products/${id}`);
+  const navigate = useNavigate();
+  const { loading, data: product, error } = useFetch(`products/${id}`);
 
-  // Handle error
-  if (error) {
-    return <div>Error: {error.message}</div>;
-  }
+  if (loading) return <div class="loading">Loading...</div>;
+  if (error) return <div>{error.message}</div>;
 
-  // Render loading indicator if data is still loading
-  if (isLoading || !product) {
-    return <LoadingIndicator />;
-  }
+  const backtoproducts = () => {
```

Comment:
use camelCase for all of your function names in Javascript.

```suggestion
  const backToProducts = () => {
```